### PR TITLE
Update File description for runtime binaries ( host,hostfxr and hostpolicy ) 

### PR DESCRIPTION
**Pull request by rakeshsinghranchi**
Fixes https://github.com/dotnet/core-setup/issues/2817

### CODE REVIEW

**Comment by rakeshsinghranchi** [comment]
@steveharter and @leecow   - PTAL . Thanks.

**Comment by leecow** [comment]
Can you give an example of what this change will look like to the user?

**Comment by rakeshsinghranchi** [comment]
For Daily build: 
.NET Core Runtime - 2.1.0-preview2-26211-0
.NET Core Host Resolver - 2.1.0-preview2-26211-0
.NET Core Runtime - 2.1.0-preview2-26211-0

For RTM : 
.NET Core Runtime - 2.1.0
.NET Core Host Resolver - 2.1.0
.NET Core Runtime - 2.1.0

**Comment by rakeshsinghranchi** [comment]
dotnet.exe
filedescription_dotnet

hostfxr.dll
filedescription_host

hostpolicy.dll
filedescription_hostpolicy

**Comment by rakeshsinghranchi** [comment]
cc @steveharter

**Comment by danmosemsft** [comment]
Is that what we call dotnet.exe, ".NET Core Runtime" ? That isn't our concept of a runtime, it's more like an enhanced host?

**Comment by Petermarcu** [comment]
I agree with Dan, dotnet.exe may be better as ".NET Core Host", which loads the .NET Core Version Selector" and the .NET Core Host Policy", which selects and loads the runtime. Doesn't seem right to call the host, the runtime because it could load any number of runtimes, sdk's and versions of them. The versions will not match the runtime in many cases and will confuse.

**Comment by rakeshsinghranchi** [comment]
@leecow / @steveharter - can you look at the updated change and approve again. Thanks.

File: src/corehost/build.proj [hunk 1]
```diff
@@ -55,10 +55,19 @@
 
     <!-- Generate Version files -->
     <ItemGroup>
-      <HostFiles Include="dotnet;hostfxr;hostpolicy" />
+      <HostFiles Include="dotnet">
+       <AssemblyName>.NET Core Host</AssemblyName>
```
Comment (weshaggard) [inline src/corehost/build.proj]: Is assembly name the correct name for this metadata? It looks more like a description. The name might cause some confusion.

**Comment by leecow** [comment]
I agree that the description for dotnet.exe should be ".NET Host". The others look good.

**Comment by rakeshsinghranchi** [comment]
>I agree that the description for dotnet.exe should be ".NET Host". The others look good.

@leecow , Just to confirm File description is ".NET **Core** Host"

**Comment by leecow** [comment]
Yes, sorry - ".NET Core Host"

**Pull request merged**